topic2/src/Persona.java

```diff
@@ -0,0 +1,11 @@
+public class Persona {
+    protected String prenume = "Sandu";
```

Comment:
propun acum sa stergi atribuirea unui valori hardcoded si sa le setezi in constructor